### PR TITLE
Minor fixes for text and styles

### PR DESCRIPTION
**Pull request by ishangupta-ds**
## Proposed changes

* Colour fixes
* text fixes

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- None

## Special notes for your reviewer:
Required for 2.0 beta 0

### CODE REVIEW

**Review by amityt — APPROVED** [review]

File: litmus-portal/frontend/src/pages/MyHub/customMyHubCard.tsx [hunk 1]
```diff
@@ -141,8 +141,8 @@ const CustomMyHubCard: React.FC<customMyHubCardProp> = ({
           src={`./icons/${
             hub.HubName === 'Chaos Hub'
               ? 'myhub-litmus.svg'
-              : hub.HubName === 'Kubera Chaos Hub'
-              ? 'kubera-chaos-hub.svg'
+              : hub.HubName === 'Litmus Chaos Hub'
```
Comment (Jonsy13) [inline litmus-portal/frontend/src/pages/MyHub/customMyHubCard.tsx:144]: @ishangupta-ds  This condition won't be needed. Only two logos are needed one for Chaos hub and one default one.
cc: @amityt 
Only one condition should do the work like below.
```suggestion
              hub.HubName === 'Chaos Hub'
              ? 'myhub-litmus.svg': 'mu-hub-charts.svg'

```

**Review by Jonsy13 — CHANGES_REQUESTED** [review]
@ishangupta-ds PTAL

**Review by amityt — DISMISSED** [review]

**Comment by ishangupta-ds** [comment]
> @ishangupta-ds PTAL

updated PTAL @Jonsy13

**Review by Jonsy13 — DISMISSED** [review]

**Review by SarthakJain26 — APPROVED** [review]

**Review by amityt — APPROVED** [review]

**Review by Jonsy13 — APPROVED** [review]

**Review by SarthakJain26 — APPROVED** [review]

**Review by whitetiger1399 — APPROVED** [review]
Awesome

**Review by rajdas98 — APPROVED** [review]

**Pull request merged**